girder/web_client/src/stylesheets/widgets/userOtpManagementWidget.styl

```diff
@@ -146,7 +146,7 @@
               &:before
                 display none
 
-@media screen and (max-width:960px)
+@media screen and (max-width960px)
```

Comment:
Is this correct syntax?

Reply:
Hmm  -- I just ran with the --fix option.  Let me check.

Reply:
Oddly, it is valid.  But I made a push to make it more consistent anyway.